### PR TITLE
Added descriptions of output configurations

### PR DESCRIPTION
**Pull request by lpaulmp**
Patch for the issue https://github.com/fpco/terraform-aws-foundation/issues/57

### CODE REVIEW

**Comment by ketzacoatl** [comment]
Minor edits to make:
* drop the leading space, there is `description = " foo"` instead of `description = "foo"`
* whitespace alignment (see `tf fmt`), eg:
 
```
variable "ssh_pubkey" {
  default     = "~/.ssh/id_rsa.pub"
  description = "file path to the pubkey to use for SSH and AWS keypair"
}
```

File: examples/kube-stack-private/variables.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ variable "instance_type" {
 }
 
 variable "coreos_stable_ami_id" {
-  description = "set a stable coreos ami id"
+  description = "set a stable coreos ami id from https://goo.gl/2vJs7F"
```
Comment (ketzacoatl) [inline examples/kube-stack-private/variables.tf]: I would recommend http://stable.release.core-os.net/amd64-usr/current/coreos_production_ami_all.json as a more future-proof URL over http://stable.release.core-os.net/amd64-usr/1465.7.0/coreos_production_ami_hvm.txt

Comment (lpaulmp) [reply examples/kube-stack-private/variables.tf]: Sure I'm updating the link

**Comment by lpaulmp** [comment]
@ketzacoatl Done can you check, please?

**Pull request merged**